SocialCareCaseViewerApi/V1/Controllers/SocialCareCaseViewerApiController.cs

```diff
@@ -127,6 +127,25 @@ public IActionResult ListCases([FromQuery] ListCasesRequest request)
             }
         }
 
+        /// <summary>
+        /// Find specific case by unique Record ID produced by MongoDB
+        /// </summary>
+        /// <response code="200">Success. Returns case related to the specified ID</response>
+        /// <response code="404">No cases found for the specified ID or officer email</response>
+        [ProducesResponseType(typeof(CareCaseDataList), StatusCodes.Status200OK)]
+        [HttpGet]
+        [Route("cases/{recordId}")]
+        public IActionResult GetCaseByRecordId(string recordId)
```

Comment:
Boundary object would perhaps be better here even though it'll have only one property. Please add validation to check that the string is valid 24 digit hex string.

Reply:
validation added